frontend/src/bundles/studio/components/timeline/timeaxis/timeaxis.tsx

```diff
@@ -0,0 +1,117 @@
+import { useTimelineContext } from 'dnd-timeline';
+
+import { Box } from '~/bundles/common/components/components.js';
+import { useMemo } from '~/bundles/common/hooks/hooks.js';
+
+type Marker = {
+    label?: string;
+    sideDelta: number;
+    heightMultiplier: number;
+};
+
+type MarkerDefinition = {
+    value: number;
+    maxRangeSize?: number;
+    minRangeSize?: number;
+    getLabel?: (time: Date) => string;
+};
```

Comment:
lets move this to a separate files and leave only Properties here

Reply:
Fixed